cognite/client/utils/_time.py

```diff
@@ -120,11 +124,6 @@ def granularity_to_ms(granularity: str) -> int:
     return ms
 
 
-def granularity_unit_to_ms(granularity: str) -> int:
-    granularity = re.sub(r"^\d+", "1", granularity)
-    return granularity_to_ms(granularity)
```

Comment:
With input of something like `"30x"` when `x` is an invalid unit, or `"23"` (missing time unit), the error message would say that the input was `"1x"` or just `"1"`